src/rooms/rooms.controller.ts

```diff
@@ -23,6 +24,8 @@ export class RoomsController {
         return this.roomsService.getRoomById(id);
     }
 
+    @ApiBearerAuth()
+    @UseGuards(AuthGuard('jwt'))

```

Comment:
Do role check of user (hotel ownerid , admin role on roles)

Reply:
ok